### PR TITLE
removing boilerplate code

### PR DESCRIPTION
**Pull request by conorhutchins**
removing boilerplate code from the automatically generated files. Left some of the comments incase they were helpful in time to come but hope to have cut much of the unnecessary lines/ files

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# https://www.robotstxt.org/robotstxt.html
```
Comment (WillMayger) [inline public/robots.txt:1]: Did this one need to be removed?

File: public/index.html [hunk 2]
```diff
@@ -5,11 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
```
Comment (WillMayger) [inline public/index.html:10]: This could probably just have had the content updated rather than be removed

File: public/index.html [hunk 3]
```diff
@@ -24,10 +20,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```
Comment (WillMayger) [inline public/index.html:27]: Same for this, did this need to be removed instead of updated?

File: public/index.html [hunk 4]
```diff
@@ -24,10 +20,8 @@
...
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (WillMayger) [inline public/index.html:30]: This boilerplate (`noscript`) can be useful to those who don't have JS enabled in their browsers (although that is very rare), did it need to be removed?

**Review by WillMayger — COMMENTED** [review]
Nice one, a few comments from me :)

**Comment by conorhutchins** [comment]
Cheers for your thoughts on this one, I've brought back the files and made the adaptions. Few things learnt already

File: public/index.html [hunk 5]
```diff
@@ -24,10 +23,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Weather App</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (WillMayger) [inline public/index.html:30]: This one here could do with being added back in, but also its not essential for this task

**Review by WillMayger — APPROVED** [review]
Nice one!

**Pull request merged**